### PR TITLE
New package: Corel.MindManager24 version 24.0.181.0

### PR DESCRIPTION
**Pull request by hmlervik**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/179824)

### CODE REVIEW

**Comment by hmlervik** [comment]
@microsoft-github-policy-service agree company="Intility"

**Comment by hmlervik** [comment]
https://en.wikipedia.org/wiki/MindManager#:~:text=On%20August%209%2C%202016%2C%20Corel%20announced%20that%20they%20had%20acquired%20the%20Mindjet%20MindManager%20business.%5B19%5D

https://blog.mindmanager.com/mindmanager-joins-corel-2/

File: manifests/c/Corel/MindManager24/24.0.181.0/Corel.MindManager24.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Corel.MindManager24
+PackageVersion: 24.0.181.0
+PackageLocale: en-US
+Publisher: Corel Corporation
+PackageName: MindManager 24
```
Comment (stephengillie) [inline manifests/c/Corel/MindManager24/24.0.181.0/Corel.MindManager24.locale.en-US.yaml:8]:
```suggestion
PackageName: MindManager 24
PackagerUrl: https://www.mindmanager.com/en/product/mindmanager/
```

**Review by stephengillie — COMMENTED** [review]
I came across an EXE installer, with a slightly different subdomain name: `https://dwnld.mindjet.com/stubs/Release/MindManager24/MindManager24_setup.exe`.

**Comment by stephengillie** [comment]
Verify:
image
The application doesn't allow interaction without registration, so I couldn't get to the version number.
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**